code/src/components/Album.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+
+import Images from "./Images";
+import AlbumTitle from "./AlbumTitle";
+import Artist from "./Artist";
+
+const Album = (props) => {
```

Comment:
What do you think about using destructuring props in React? Maybe, It's a good option to use it in this case where we need to use many properties like image, URL, name, and so on 😄